apps/api/src/config/index.ts

```diff
@@ -236,9 +236,15 @@ export const config = {
     isEnabled: process.env.INDEXING_ENABLED === "true",
     stakingContract: process.env.INDEXING_STAKING_CONTRACT,
     rewardsContract: process.env.INDEXING_REWARDS_CONTRACT,
-    startBlock: process.env.INDEXING_START_BLOCK
-      ? parseInt(process.env.INDEXING_START_BLOCK, 10)
+    convictionClaimsContract:
+      process.env.INDEXING_CONVICTION_CLAIMS_CONTRACT ||
+      "0x6A3044c1Cf077F386c9345eF84f2518A2682Dfff",
```

Comment:
what is this default? should it be here?

Reply:
The default is the mainnet airdrop contract.  Maybe makes sense to remove it here, and leave it in the example env?